client/src/pages/Client/AllProgrammes/index.js

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import { Row, Col } from '../../../components/Grid';
+import { Programmes } from '../../../api-calls';
+import { Basic, Link } from '../../../components/Cards';
+import Title from '../../../components/Title';
+import { navRoutes } from '../../../constants';
+
+import * as T from '../../../components/Typography';
+
+const introText = `Here you will find the weekly resources that your therapist has prepared especially for you. You can access these resources in between sessions to enhance your therapeutic outcomes.`;
+
+const AllProgrammes = () => {
+  const [programmes, setProgrammes] = useState([]);
+
+  useEffect(() => {
+    // put in api call to get the programme data
+    const getProgrammes = async () => {
+      const { data, error } = await Programmes.getProgrammesByClient();
+      if (!error) {
+        console.log('DARTA!', data);
+        setProgrammes(data);
+      }
+    };
+
+    getProgrammes();
+  }, []);
+
+  const programmesToView = programmes.length > 0;
+
+  return (
+    <>
```

Comment:
the width of cards and the paragraph doesn't match the wireframe, something I've faced on my PR too 
![image](https://user-images.githubusercontent.com/39485495/113567792-f09a7280-9617-11eb-8da5-3221737a2ca5.png)
my solution was but now we should make  common solution for this 
```suggestion
    <div style={{maxWidth: 1065}}>
```

Reply:
It's fine that the cards and paragraph change width slightly (it doesn't need to always be exactly 300px as that's not always ideal for a responsive design). I agree that there should be a max-width on the content in the layout though for extra wide screens (that was already meant to be in there from Figma as I remember discussing this with Jem but maybe was missed )